ynput/OpenPype
# Houdini: Camera Loader fix mismatch for Maya cameras

## Changelog Description
This PR adds 
- A workaround to match Maya render mask in Houdini
- `SetCameraResolution` inventory action
- set camera resolution when loading or updating camera

![image](https://github.com/ynput/OpenPype/assets/20871534/524686b9-db3d-4af5-81bc-d84d00c362ef)

## Additional Info 1
Many thanks to @BigRoy, Code in this PR is mostly his..
https://github.com/BigRoy/OpenPype/blob/2810ee866066b55ad7f26ab2cebf2dc6de8978f2/openpype/hosts/houdini/plugins/load/load_camera.py

## Additional Info 2
This PR also fixes this issue [Set Camera resolution for Loaded camera in Houdini](https://community.ynput.io/t/set-camera-resolution-for-loaded-camera-in-houdini/412) (it was posted to forums)

## Testing notes:
1. Load alembic camera in houdini
2. Set version from `Manage` and check Aperture and Resolution
3. Test Inventory action `Set Camera Resolution`
